### ISSUE TITLE
Feature Request: optionally alerting users from any app when they've been followed

### ISSUE THREAD

**Issue by thecurtic** [damus-io/damus#2576]
## Have a go at filling out the User Story template below

As a Damus user who is _____________, I would like to _________________, so that I achieve ___________.

**Is your feature request related to a problem? Please describe.**
The problem is that when following another user of whom I'm in the physical presence, we are both stuck with scanning each other's QR codes to find each other on nostr. 

** When does this problem happen? **
Any time I try exchanging npubs in person

**Describe the solution you'd like**
When I tap "follow" on a nostr profile using Damus, I'd like to see an option (tick box or otherwise) to send that user a DM from my user profile which tells them that I have followed them.

This gives them the option to follow back in the moment or at another time, which means they won't have to scan my QR code right then and there.

**Describe alternatives you've considered**
No alternatives, but perhaps you have other ideas that may be even more efficient.

**Additional context**

**Comment by alltheseas** [comment]
> When I tap "follow" on a nostr profile using Damus, I'd like to see an option (tick box or otherwise) to send that user a DM from my user profile which tells them that I have followed them.

After you have followed someone, have you considered sending them a DM from profile view? 

![image](https://github.com/user-attachments/assets/1cb2fb9f-376a-4802-afef-583b0646b68c)

**Comment by thecurtic** [comment]
Yes, but the problem is moreso that the app doesn't prompt the user to do this. I think it would smooth out the experience.

**Comment by alltheseas** [comment]
Perhaps 'notify me if someone follows me' https://github.com/damus-io/damus/issues/991 would be a better solution to your problem statement?

**Comment by thecurtic** [comment]
is it possible for damus to detect that reliably?
